scripts/stats.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../db.json');
```

Comment:
I would suggest moving this into a separate PR, as it's a separate change from adding the mime-support upstream.

Reply:
Done.